include/cloudbreak.bash

```diff
@@ -146,42 +148,50 @@ cloudbreak-conf-db() {
     env-import CB_DB_ENV_PASS ""
     env-import CB_DB_ENV_SCHEMA "public"
     env-import CB_HBM2DDL_STRATEGY "validate"
+    env-import CB_DB_PORT_5432_TCP_ADDR "$COMMON_DB"
 
     env-import PERISCOPE_DB_ENV_USER "postgres"
     env-import PERISCOPE_DB_ENV_DB "periscopedb"
     env-import PERISCOPE_DB_ENV_PASS ""
     env-import PERISCOPE_DB_ENV_SCHEMA "public"
     env-import PERISCOPE_HBM2DDL_STRATEGY "validate"
+    env-import PERISCOPE_DB_PORT_5432_TCP_ADDR "$COMMON_DB"
```

Comment:
The env variable name is different for Consumption service.
What is the intent?